src/ontology/mappings/fbbt-mappings.sssom.tsv

```diff
@@ -2,447 +2,445 @@
-FBbt:00017021	abdominal histoblast anlage	semapv:crossSpeciesExactMatch	UBERON:6017021	semapv:ManualMappingCuration
-FBbt:00040003	non-connected developing system	semapv:crossSpeciesExactMatch	UBERON:6040003	semapv:ManualMappingCuration
-FBbt:00040005	synaptic neuropil	semapv:crossSpeciesExactMatch	UBERON:6040005	semapv:ManualMappingCuration
-FBbt:00040007	synaptic neuropil domain	semapv:crossSpeciesExactMatch	UBERON:6040007	semapv:ManualMappingCuration
-FBbt:00041000	synaptic neuropil block	semapv:crossSpeciesExactMatch	UBERON:6041000	semapv:ManualMappingCuration
-FBbt:00047153	anus	semapv:crossSpeciesExactMatch	UBERON:0001245	semapv:ManualMappingCuration
-FBbt:00048032	glycinergic neuron	semapv:crossSpeciesExactMatch	CL:1001509	semapv:ManualMappingCuration
-FBbt:00057001	anterior-posterior subdivision of organism	semapv:crossSpeciesExactMatch	UBERON:6057001	semapv:ManualMappingCuration
-FBbt:00057012	female germline cell	semapv:crossSpeciesExactMatch	CL:0000025	semapv:ManualMappingCuration
-FBbt:00058020	juvenile hormone secreting cell	semapv:crossSpeciesExactMatch	CL:0000482	semapv:ManualMappingCuration
-FBbt:00058291	dorsal vessel	semapv:crossSpeciesExactMatch	UBERON:0015231	semapv:ManualMappingCuration
-FBbt:00100152	row	semapv:crossSpeciesExactMatch	UBERON:0034926	semapv:ManualMappingCuration
-FBbt:00100153	sensillum row	semapv:crossSpeciesExactMatch	UBERON:6100153	semapv:ManualMappingCuration
-FBbt:00100291	glutamatergic neuron	semapv:crossSpeciesExactMatch	CL:0000679	semapv:ManualMappingCuration
-FBbt:00100313	multicellular structure	semapv:crossSpeciesExactMatch	UBERON:0010000	semapv:ManualMappingCuration
-FBbt:00100314	duct	semapv:crossSpeciesExactMatch	UBERON:0000058	semapv:ManualMappingCuration
-FBbt:00100315	gut section	semapv:crossSpeciesExactMatch	UBERON:0004921	semapv:ManualMappingCuration
-FBbt:00100316	diverticulum of gut	semapv:crossSpeciesExactMatch	UBERON:0009854	semapv:ManualMappingCuration
-FBbt:00100317	gland	semapv:crossSpeciesExactMatch	UBERON:0002530	semapv:ManualMappingCuration
-FBbt:00110636	adult cerebral ganglion	semapv:crossSpeciesExactMatch	UBERON:6110636	semapv:ManualMappingCuration
-FBbt:00110746	presumptive prothoracic metatarsus	semapv:crossSpeciesExactMatch	UBERON:6110746	semapv:ManualMappingCuration
-FBbt:00110811	presumptive arista	semapv:crossSpeciesExactMatch	UBERON:6110811	semapv:ManualMappingCuration
-FBbt:10000000	anatomical entity	semapv:crossSpeciesExactMatch	UBERON:0001062	semapv:ManualMappingCuration
+#mapping_set_id: "http://purl.obolibrary.org/obo/fbbt/fbbt-mappings.sssom.tsv"
+#mapping_set_description: "Mappings between the Drosophila Anatomy Ontology and foreign ontologies."
+#creator_id:
+#  - "https://orcid.org/0000-0002-6095-8718"
+#license: "https://creativecommons.org/licenses/by/4.0/"
+#subject_source: "http://purl.obolibrary.org/obo/fbbt.owl"
+#mapping_date: "2023-08-09"
```

Comment:
@gouttegd do you confirm these changes, please?

Reply:
Yes, everything is normal here. :)

The entire file has changed because we’ve added `object_source` to all rows.

Reply:
Did you notice no more "semapv" in the curie_mapie?

Reply:
As an aside, I’m starting to wonder whether it really makes sense to have this file committed in the repository, given that it’s a 100% exact copy of the file published by FBbt…

The only benefit of having this file here is that we don’t need network access to fetch it at release time, but given that we already have several network dependencies for a release, I am not entirely convince this is worth it.

Anyway, that’s not something to be changed right now while we are on the verge of releasing. Just something to note for later.

Reply:
smart this SSSOM haha